test: Intermittent failure in feature_config_args

What looks like an intermittent failure in `feature_config_args.py` happened in CI testing of #21407.

https://cirrus-ci.com/task/5952013763608576

```
 test  2021-03-16T11:11:46.877000Z TestFramework (ERROR): Assertion failed 
                                   Traceback (most recent call last):
                                     File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_framework.py", line 128, in main
                                       self.run_test()
                                     File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/feature_config_args.py", line 219, in run_test
                                       self.test_seed_peers()
                                     File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/feature_config_args.py", line 214, in test_seed_peers
                                       self.nodes[0].setmocktime(start + 65)
                                     File "/usr/lib/python3.6/contextlib.py", line 88, in __exit__
                                       next(self.gen)
                                     File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_node.py", line 400, in assert_debug_log
                                       self._raise_assertion_error('Expected messages "{}" does not partially match log:\n\n{}\n\n'.format(str(expected_msgs), print_log))
                                     File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_node.py", line 166, in _raise_assertion_error
                                       raise AssertionError(self._node_msg(msg))
                                   AssertionError: [node 0] Expected messages "['Adding fixed seeds as 60 seconds have passed and addrman is empty']" does not partially match log:
                                    - 2021-03-16T11:10:26.849988Z (mocktime: 2021-03-16T11:10:26Z) [httpserver.cpp:238] [http_request_cb] Received a POST request for / from 127.0.0.1:35268
                                    - 2021-03-16T11:10:26.850095Z (mocktime: 2021-03-16T11:10:26Z) [rpc/request.cpp:174] [parse] ThreadRPCServer method=setmocktime user=__cookie__
                                    - 2021-03-16T11:10:26.853443Z (mocktime: 2021-03-16T11:11:31Z) [util/system.h:468] [TraceThread] msghand thread start
                                    - 2021-03-16T11:10:26.853489Z (mocktime: 2021-03-16T11:11:31Z) [util/system.h:468] [TraceThread] opencon thread start
                                    - 2021-03-16T11:10:26.853541Z (mocktime: 2021-03-16T11:11:31Z) [util/system.h:468] [TraceThread] addcon thread start
                                    - 2021-03-16T11:10:26.853580Z (mocktime: 2021-03-16T11:11:31Z) [net.cpp:394] [ConnectNode] trying connection fakenodeaddr lastseen=0.0hrs
                                    - 2021-03-16T11:10:26.854222Z (mocktime: 2021-03-16T11:11:31Z) [util/system.h:468] [TraceThread] net thread start
                                    - 2021-03-16T11:11:26.797916Z (mocktime: 2021-03-16T11:11:31Z) [random.cpp:523] [SeedPeriodic] Feeding 14413 bytes of dynamic environment data into RNG
                                    - 2021-03-16T11:11:27.362393Z (mocktime: 2021-03-16T11:11:31Z) [net.cpp:394] [ConnectNode] trying connection fakenodeaddr lastseen=0.0hrs
```
It  is expecting the messsage "Adding fixed seeds as 60 seconds have passed and addrman is empty" but is not getting it.

--- a/test/functional/feature_config_args.py
+++ b/test/functional/feature_config_args.py
@@ -165,6 +165,7 @@ class ConfArgsTest(BitcoinTestFramework):
         with self.nodes[0].assert_debug_log(expected_msgs=[
                 "Loaded 0 addresses from peers.dat",
                 "0 addresses found from DNS seeds",
+                "opencon thread start",  # Ensure ThreadOpenConnections::start time is properly set
         ]):
             self.start_node(0, extra_args=['-dnsseed=1', '-fixedseeds=1', f'-mocktime={start}'])
         with self.nodes[0].assert_debug_log(expected_msgs=[
@@ -206,6 +207,7 @@ class ConfArgsTest(BitcoinTestFramework):
         with self.nodes[0].assert_debug_log(expected_msgs=[
                 "Loaded 0 addresses from peers.dat",
                 "DNS seeding disabled",
+                "opencon thread start",  # Ensure ThreadOpenConnections::start time is properly set
         ]):
             self.start_node(0, extra_args=['-dnsseed=0', '-fixedseeds=1', '-addnode=fakenodeaddr', f'-mocktime={start}'])
         with self.nodes[0].assert_debug_log(expected_msgs=[

